ScienceJournal/UI/ObserveViewController.swift

```diff
@@ -884,6 +884,10 @@ open class ObserveViewController: ScienceJournalCollectionViewController, ChartC
           UIView.layoutFittingCompressedSize).height : 0
       collectionView?.contentInset.top = topInset
       collectionView?.scrollIndicatorInsets.top = topInset
+      if collectionView?.contentOffset.y == 0 {
+        // Adjusts the offset so the timeAxisView doesn't obscure the top-most card
+        collectionView?.contentOffset.y = -topInset
+      }
```

Comment:
This should probably be flagged, so the scroll position doesn't move when the AA feature is disabled.